docs/diga_api.md

```diff
@@ -0,0 +1,216 @@
+# Overview
+
+Manufacturers of digitial health applications (DiGA) can apply for a fast track at [bfarm](https://www.bfarm.de/EN/MedicalDevices/DiGA/_node.html) to place their solutions in a directory of apps which can be prescribed by doctors to patients. By showing evidence that the apps improve patients health and by obtaining medical device certifications, it can be assured that the apps are of high quality. On the other side, DiGA manufacturers get reimbursed by health insurances when their apps are prescribed to patients. The prescription process is build around a 16-character prescription code which can be used to activate the apps but which is also required to send the invoices. This repository holds code to:
+
+- verify that user-entered prescription codes are valid
+- create and send invoices to insurance companies based on prescription codes.
+
+This documentation complements the code by describing the DiGA api and the full flow on a higher level. With this documentation we hope that other DiGA manufacturers can understand the required steps to verify and reimburse prescription codes without having to read through the (mainly German) official documentation. Also we hope that it will facilitate writing similar solutions in other programming languages if necessary. Finally, it can be used as a place to collect questions and answers around the DiGA api.
+
+## Summary
+
+- user enters prescription code in the DiGA (app)
+- insurance information is extracted from the code
+- a request is made to the api of the insurance to verify the code
+  - response contains `Tag der Leistungserbringung` which is the day a user started to use the app
+  - this is required for the billing process
```

Comment:
What does `this` refer to? 
Perhaps clearer if it is just

> o use the app and is required for the billing process

Reply:
Refers to the response value of `Tag der Leistungserbringung`. Will clarify